src/styles/body.scss

```diff
@@ -0,0 +1,9 @@
+
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
```

Comment:
The `position: relative;` property on the `<body>` element is unusual. Unless you have child elements that are positioned absolutely in relation to the body, this rule can be omitted.